lib/geordi/settings.rb

```diff
@@ -114,7 +114,7 @@ def save_global_settings
     # deprecated
     def gitpt_api_key_old
       file_path = File.join(ENV['HOME'], '.gitpt')
-      if File.exist?(file_path)
+      if File.exist?(file_path) && !Util.testing?
```

Comment:
Ok. It is hacky to disable the method this way, but there is no way around – the test cannot rely on files in the developer's `~`, and after all, it is deprecated.